src/app/screens/History/ExpressionContainer/index.js

```diff
@@ -30,4 +77,8 @@ const ExpressionContainer = ({expression}) => {
     </View>
   );
 };
-export default ExpressionContainer;
+const mapDispatchToProps = {
+  editExpressionHistory: actionHistory.editExpressionHistory,
+  deleteHistoryForId: actionHistory.deleteHistoryForId,
+};
```

Comment:
Don't use mapDispatchToProps. Remember the redux talk Seba made. It's not that good of a pattern and it makes more code.